python/trainer_config.yaml

```diff
@@ -114,10 +114,11 @@ VisualPyramidBrain:
 
 Ball3DBrain:
     normalize: true
-    batch_size: 1200
+    batch_size: 64
```

Comment:
Does this really work better ?

Reply:
Yep it does. I have some crazy theories. We can talk through it more when you are back.